pdns/dnsdist-web.cc

```diff
@@ -128,7 +128,7 @@ static bool isAnAPIRequestAllowedWithWebAuth(const YaHTTP::Request& req)
 
 static bool isAStatsRequest(const YaHTTP::Request& req)
 {
-  return req.url.path == "/jsonstat";
+  return req.url.path == "/jsonstat" || req.url.path == "/prometheus";
```

Comment:
Really looking forward for this to be merged!!! 👍 🙌 

This is not really a big deal, but Prometheus default to `/metrics` for metrics path, would be nice to stick to the standard `/metrics` unless there's a reason no to.

```
# The HTTP resource path on which to fetch metrics from targets.
[ metrics_path: <path> | default = /metrics ]
```
https://prometheus.io/docs/prometheus/latest/configuration/configuration/